package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "metal-myComponent",
+	"version": "0.0.0",
+	"description": "My awesome Metal.js project",
+	"license": "BSD",
+	"repository": "zxdgoal/metal-myComponent",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "src/MyComponent.js",
+	"files": [
+		"src",
```

Comment:
Make sure to also add `lib` to the list of files that will be published to npm here, just like in [metal-modal's package.json](https://github.com/metal/metal-modal/blob/master/package.json#L18).